resources/i18n/en.json

```diff
@@ -125,5 +125,87 @@
+    "item.metadata.dc.title.minLength": "Item metadata dc.title must have at least 4 characters.",
+    "item.metadata.dc.title.maxLength": "Item metadata dc.title must have at most 128 characters.",
+    
+    "item.metadata.dc.type.required": "Item metadata dc.type is required.",
+    "item.metadata.dc.type.minLength": "Item metadata dc.type must have at least 3 characters.",
+    "item.metadata.dc.type.maxLength": "Item metadata dc.type must have at most 128 characters.",
+    
+    "item.metadata.dc.date.created.required": "Item metadata dc.date.created is required.",
+    
+    "item.metadata.dc.language.iso.required": "Item metadata dc.language.iso is required.",
+    
+    "item.metadata.dc.contributor.author.required": "Item metadata dc.contributor.author is required.",
+    "item.metadata.dc.contributor.author.minLength": "Item metadata dc.contributor.author must have at least 4 characters.",
+    "item.metadata.dc.contributor.author.maxLength": "Item metadata dc.contributor.author must have at most 128 characters.",
+    
+    "item.metadata.dc.description.required": "Item metadata dc.description is required.",
+    "item.metadata.dc.description.minLength": "Item metadata dc.description must have at least 8 characters.",
+    "item.metadata.dc.description.maxLength": "Item metadata dc.description must have at most 512 characters.",
+    
+    "item.metadata.dc.description.abstract.required": "Item metadata dc.description.abstract is required.",
+    "item.metadata.dc.description.abstract.minLength": "Item metadata dc.description.abstract must have at least 16 characters.",
+    "item.metadata.dc.description.abstract.maxLength": "Item metadata dc.description.abstract must have at most 2056 characters.",
+    
+    "item.metadata.dc.subject.required": "Item metadata dc.subject is required.",
+    "item.metadata.dc.subject.minLength": "Item metadata dc.subject must have at least 4 characters.",
+    "item.metadata.dc.subject.maxLength": "Item metadata dc.subject must have at most 128 characters.",
+    
+    "item.metadata.dc.rights.required": "Item metadata dc.rights is required.",
+    "item.metadata.dc.rights.minLength": "Item metadata dc.rights must have at least 4 characters.",
+    "item.metadata.dc.rights.maxLength": "Item metadata dc.rights must have at most 128 characters.",
+    
+    "item.metadata.dc.rights.uri.required": "Item metadata dc.rights.uri is required.",
+    "item.metadata.dc.rights.uri.minLength": "Item metadata dc.rights.uri must have at least 4 characters.",
+    "item.metadata.dc.rights.uri.maxLength": "Item metadata dc.rights.uri must have at most 512 characters.",
+    "item.metadata.dc.rights.uri.pattern": "Item metadata dc.rights.uri must be a FQDN.",
+    
+    "item.create.processing": "Item {{name}} is being created. Please wait.",
+    "item.create.success": "Item {{name}} was created successfully.",
+    "item.create.error": "Error creating item {{name}}.",
```

Comment:
@DylanMeeus : this seems to accidentally duplicate all the Item metadata/create language tags in `en.json`.  This tags were all moved up recently to reorder the `en.json` alphabetically.